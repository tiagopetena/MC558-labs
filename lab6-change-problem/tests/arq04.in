25 2998
91 397 1
94 252 3
2 866 1
17 333 1
36 613 1
68 466 3
78 435 1
73 913 2
39 51 2
11 551 1
67 254 2
1 310 1
54 705 1
99 975 3
83 389 2
100 322 3
26 166 2
56 419 2
74 741 1
44 937 2
15 78 3
13 363 2
9 857 3
7 608 3
87 250 1
