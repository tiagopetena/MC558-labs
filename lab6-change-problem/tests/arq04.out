2429 22859
